contracts/state/State.sol

```diff
@@ -452,4 +452,13 @@ contract State is Ownable2StepUpgradeable, IState {
         _defaultIdType = defaultIdType;
         _defaultIdTypeInitialized = true;
     }
+
+    /**
+     * @dev Check if the identity is managed by the msg.sender
+     * @param id Identity
+     * @return True if the identity is managed by the msg.sender
+     */
+    function _isIdManagedByMsgSender(uint256 id) internal view returns (bool) {
```

Comment:
I think function name should somehow describe that it's for Ethereum controlled identities.
Something like _isEthIDOwner/Controller.

Reply:
hm, it may look puzzling as `id` is not owner or controller itself by rather controlled by something. What about `_isIdControlledByEth`?